labextension/src/index.ts

```diff
@@ -105,6 +175,15 @@ async function activate(
     },
   });
 
+  commands.addCommand(CommandIDs.open, {
+    execute: (args) => {
```

Comment:
recommend typing the args beyond `any`, and making them more descriptive.

Reply:
This is done in #425 and once we rebase onto `main` it will be addressed